hyper/http20/connection.py

```diff
@@ -122,17 +122,17 @@ def __init_state(self):
 
         # Values for the settings used on an HTTP/2 connection.
         self._settings = {
-            SettingsFrame.INITIAL_WINDOW_SIZE: 65535,
+            SettingsFrame.INITIAL_WINDOW_SIZE: 2**24,
```

Comment:
This setting keeps track of the *remote* side's flow control window, so we can't just change it.